app/renderer/js/utils/dnd-util.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const ConfigUtil = require(__dirname + '/config-util.js');
+
+function dnd() {
+	this.initDNDButton();
```

Comment:
This is from the old commit which should be removed!

Reply:
I think, we need to initialize the button.

Reply:
Yeah, so you need to do this here :)